### ISSUE TITLE
KNXNetIP driver generating excessive promise count 

### ISSUE THREAD

**Issue by AndersHogqvist** [pixilab/blocks-script#21]
We use the KNXNetIP driver to control the lights in our facility but there seems to be a problem with it that I haven't been able to solve. It's constantly printing "Script: 'Possibly excessive promise count' from driver/KNXNetIP, <incrementing number>" in the Blocks log. These come in chunks with seemingly random intervals stretching from 30 seconds to several minutes with a constantly increasing number of promises. I'm looking at it right now and it's currently at 16150.

I have tried increasing the intervals in which it checks the state (the argument passed to checkStateSoon) but without any luck. Any ideas?

**Comment by TheWizz** [comment]
Please send your question to ***@***.***, and we'll take it there instead.

Mike

**Comment by AndersHogqvist** [comment]
That was an interesting email adress 😉

**Comment by TheWizz** [comment]
Yes, indeed. Persumably Github stripped it out. Another try: support AT pixilab.se
